docs/specs/config.yml

```diff
@@ -508,11 +508,11 @@ commands:
 inputs:
```

Comment:
how about a dedicated `localization.yml` spec ?

Reply:
sounds good, changed

---
In reply to: [223994665](https://github.com/dotnet/docfx/pull/3482#discussion_r223994665) [](ancestors = 223994665)